moses/moses/main/populate_atomspace.h

```diff
@@ -38,22 +38,19 @@ using namespace combo;
  * @param Atomspace&     as
  * @param Itable&     itable
  *
- * @return                 LinkValue pointer containing the logical_and.
+ * @return
```

Comment:
I think you can just delete the line `@return`, though again I don't know doxygen.

Reply:
What about the other `@return`s ? Should they be kept?

Reply:
Doing a quick regex search over the opencog repositories I found no `@return` comment on functions returning `void`. The only `@return` comments with no trailing comments, such as in `load_table.h` are not over void functions, so I think it is better to remove that one, and leave the others but again I'm no doxygen user so...